web/src/RootSwitcher.tsx

```diff
@@ -84,11 +84,8 @@ const RootSwitcher = ({ setContentLanguage }: RootSwitcherProps): ReactElement =
 
         {/* Redirects */}
         <Route path='/' element={<Navigate to={fixedCityPath ?? landingPath} replace />} />
-        <Route path={LANDING_ROUTE} element={<Navigate to={fixedCityPath ?? landingPath} replace />} />
+        <Route path={RoutePatterns[LANDING_ROUTE]} element={<Navigate to={fixedCityPath ?? landingPath} replace />} />
```

Comment:
Hmmm... While this works and there is no endless loop, we are basically redirecting from `/landing/de` to `/landing/de` if there is no fixed city. I'd prefer if we could avoid that by only having that redirect if there is a fixed city.
```suggestion
        {!!fixedCityPath && (
          <Route path={RoutePatterns[LANDING_ROUTE]} element={<Navigate to={fixedCityPath} replace />} />
        )}
```